arch/riscv/core/CMakeLists.txt

```diff
@@ -21,6 +20,7 @@ endif ()
 zephyr_library_sources_ifdef(CONFIG_FPU_SHARING fpu.c fpu.S)
 zephyr_library_sources_ifdef(CONFIG_DEBUG_COREDUMP coredump.c)
 zephyr_library_sources_ifdef(CONFIG_IRQ_OFFLOAD irq_offload.c)
+zephyr_library_sources_ifdef(CONFIG_GEN_SW_ISR_TABLE isr.S)
```

Comment:
if I read this correctly, this PR will disable building `isr.S` for all the targets not defining `CONFIG_GEN_SW_ISR_TABLE`. I understand this is needed for the `nRF54L15 FLPR` target. Since this PR doesn't enable this CONFIG for any platform, it will effectively remove the isr.S file from all the targets. Am I missing something?

Reply:
I see, thanks for clarification